cnfapp/hooks/tests.yml

```diff
@@ -16,5 +16,7 @@
     opcap_target_catalog_source: "nfv-example-cnf-catalog"
     opcap_catalog_source_namespace: "openshift-marketplace"
     opcap_output_dir: "{{ job_logs.path }}"
+  when:
+    - run_opcap_check|default(true)|bool
```

Comment:
Ramon, don't you think would be cool to make this var official to control opcap check and document it ? other than that, looks good to me

Reply:
done